projects/hslayers/src/components/print/models/text-style.model.ts

```diff
@@ -0,0 +1,12 @@
+import {xPos, yPos} from '../types/XY-positions.type';
+
+export type TextStyle = {
+  fillColor?: string;
+  strokeColor?: string;
+  textSize?: string;
+  fontFamily?: string;
+  fontStyle?: string;
+  textDraw?: 'fill' | 'stroke';
```

Comment:
Also, what is this param good for?

Reply:
textDraw I use for setting text title style drawn.